src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    static final int COLOR_NUMBER = 3;
+    private final Random random = new Random();
+
+    public Color getRandomColor() {
+        int colorNumber = random.nextInt(COLOR_NUMBER);
+        Color[] color = Color.values();
+        return color[colorNumber];
     }
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-temporary-variables-if-you-can-directly-return-value-from-the-method